qt_ui/windows/mission/flight/QFlightCreator.py

```diff
@@ -261,6 +262,24 @@ def on_squadron_changed(self, index: int) -> None:
 
     def update_max_size(self, available: int) -> None:
         self.flight_size_spinner.setMaximum(min(available, 4))
```

Comment:
I think you can just do `min(available, self.aircraft_selector.currentData().dcs_unit_type.group_size_max)` and delete the rest of the change.

Reply:
I'm unable to do that as:

`self.departure.aircraft.group_size_max))` or
`self.aircraft_selector.currentData().group_size_max))`

both operate on `AircraftType` and not `FlyingType`.  I wasn't able to get a property in `AircraftType` to access `FlyingType`.

Reply:
`AircraftType` is a wrapper around `FlyingType`. That's what `dcs_unit_type` is. Pushed a change.

According to pydcs a group of 4 KC-135s is fine though.

Reply:
Of the support aircraft, `group_size_max`  is only defined for `KJ-2000`, `KC135MPRS`, and `KC130`.

Reply:
I haven't had a chance to look at the ME. Is that wrong, or are you trying to override that behavior? I'd be surprised if it's wrong because it's autogenerated and works for many of the aircraft.

Reply:
The export is correct, however DCS' behavior with support aircraft only allows contacting the first aircraft in the flight, even if it is a flight of four.  That is why I wanted to set the default flight size to one.

See:  
[SupportAircraftBehavior.zip](https://github.com/dcs-liberation/dcs_liberation/files/6682505/SupportAircraftBehavior.zip)

![Awacs](https://user-images.githubusercontent.com/74509817/122677890-91bdb180-d1a1-11eb-8af5-bfe1c70a8320.png)
![Tankers](https://user-images.githubusercontent.com/74509817/122677893-92eede80-d1a1-11eb-8cfe-f5064f861bcd.png)

Reply:
Also am getting after making some modifications to the campaign, which is stopping the flight selection window from appearing.

[UnableToGetFlightWindow.zip](https://github.com/dcs-liberation/dcs_liberation/files/6682533/UnableToGetFlightWindow.zip)

```
Traceback (most recent call last):
  File ".\dcs_liberation\qt_ui\windows\mission\QPackageDialog.py", line 172, in on_add_flight
    self.add_flight_dialog = QFlightCreator(
  File ".\dcs_liberation\qt_ui\windows\mission\flight\QFlightCreator.py", line 96, in __init__
    self.update_max_size(self.departure.available)
  File ".\dcs_liberation\qt_ui\windows\mission\flight\QFlightCreator.py", line 265, in update_max_size
    min(available, self.departure.aircraft.dcs_unit_type.group_size_max)
AttributeError: 'NoneType' object has no attribute 'dcs_unit_type'
```

Reply:
I follow. I'll push another patch in a bit. We shouldn't be naming any aircraft explicitly in the code any more but should be using the unit data.

Reply:
Updated. Have a look. We should be customizing unit properties in the data file from now on so that we don't have to remember to specialize the behavior everywhere it's needed.